src/devices/tuya.ts

```diff
@@ -777,6 +793,30 @@ const fzLocal = {
             return tuya.fz.datapoints.convert(model, msg, publish, options, meta);
         },
     } satisfies Fz.Converter,
+    // biome-ignore lint/style/useNamingConvention: ignored using `--suppress`
+    backlight_non_relay_pos: {
+        cluster: "genOnOff",
+        type: ["attributeReport", "readResponse"],
+        convert: (model, msg, publish, options, meta) => {
+            const result: KeyValueAny = {};
+
+            if (msg.data.tuyaBacklightSwitch !== undefined) {
```

Comment:
Looks like a duplicate of `fz.backlight_mode_off_on`

Reply:
Great question! Seems it should be. Let me test.

Reply:
Issue is gone since change has been merged into `tuyaModernExtend.tuyaOnOff`